spec/models/user_spec.rb

```diff
@@ -1,5 +1,5 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
+# RSpec.describe User, type: :model do
+#   pending "add some examples to (or delete) #{__FILE__}"
+# end
```

Comment:
- [ ] This file should contain validation tests for user model. Also, any custom model in the user.rb file should also be tested.